### DISCUSSION TITLE
Snap Coloing for Osu Standard

### DISCUSSION THREAD

**Discussion by Zylviij** [ppy/osu#16597] [Ideas] [answered]
It would be awesome if there were an option to enable adding snap coloring to combo color in osu!standard. I come from a DDR / stepmania background and really enjoy using snap coloring to read rhythms, and am super excited by the addition of snap coloring to osu!mania.

I know that some mappers follow certain conventions to distinguish certain rhythms, like new-combo every note in a 1/12th stream or marking quads as a group of two doubles, but snap coloring would make these systems uniform. And in my opinion, snap coloring would look beautiful, especially on tech maps.

Not sure if this would be too OP to have as a setting as it might make high AR reading much easier, but even in that case I'd still love to have it as an unrankable mod and would use it all of the time anyways :)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/15220
https://github.com/ppy/osu/pull/15275

**Reply by Zylviij** [reply]
Ah, I did do a search of discussions and issues, but didn't find those because they don't mention the term "snap coloring"

I'll try to tag them accordingly
